src/Monolog/Handler/BrowserConsoleHandler.php

```diff
@@ -29,6 +36,9 @@ class BrowserConsoleHandler extends AbstractProcessingHandler
     /** @var FormattedRecord[] */
     protected static $records = [];
 
+    public const FORMAT_HTML = 'html';
+    public const FORMAT_JS = 'js';
```

Comment:
```suggestion
    protected const FORMAT_HTML = 'html';
    protected const FORMAT_JS = 'js';
```

I don't think there's value in this being public